website/docs/d/alert_grouping_setting.html.markdown

```diff
@@ -0,0 +1,43 @@
+---
+layout: "pagerduty"
+page_title: "PagerDuty: pagerduty_alert_grouping_setting"
+sidebar_current: "docs-pagerduty-datasource-alert-grouping-setting"
+description: |-
+  Get information about an alert grouping setting that you have created.
+---
+
+# pagerduty\_alert\_grouping\_setting
+
+Use this data source to get information about a specific [alert grouping setting][1].
+
+## Example Usage
+
+```hcl
+data "pagerduty_alert_grouping_setting" "example" {
+  name = "My example setting"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name to use to find an alert grouping setting in the PagerDuty API.
+
+## Attributes Reference
+
+* `id` - The ID of the found alert grouping setting.
+* `name` - The short name of the found alert grouping setting.
+* `description` - A description of this alert grouping setting.
+* `type` - The type of object. The value returned will be one of `content_based`, `content_based_intelligent`, `intelligent` or `time`.
+* `config` - The values for the configuration setup for this setting.
+* `services` - A list of string containing the IDs of services associated with this setting.
+
+The `config` block contains the following arguments:
+
+* `timeout` - The duration in minutes within which to automatically group incoming alerts. This setting is only required and applies when `type` is set to `time`. To continue grouping alerts until the incident is resolved leave this value unset or set it to `null`.
```

Comment:
the documentation regarding the timeout is incorrect. Apparently, the PagerDuty APIs expects the value in seconds, while the documentation says it is in minutes.

Here is the response from _terraform apply_ command:

![image](https://github.com/user-attachments/assets/e4d8106b-ba04-45e5-a3bf-26583f7719d6)